README.md

```diff
@@ -37,38 +44,44 @@ Default output format [None]: [just hit enter]
     
 
 ## Run Locally
+To run the ZotNFound app locally, follow these steps:
 
-Clone the project
+### Step 1: Clone the Repository
+Clone the repository to your local machine:
 
 ```bash
   git clone https://github.com/icssc/ZotNFound.git
 ```
 
-Go to the project directory
+### Step 2: Navigate to the Project Directory
+Change into the project directory:
 
 ```bash
   cd zotnfound
 ```
 
-Install dependencies
+### Step 3: Install Backend Dependencies
+Install the backend dependencies using pnpm:
 
 ```bash
   pnpm install
 ```
 
-Start backend 
+### Step 4: Start the Backend
+Start the backend server with:
 
 ```bash
-  cd ./packages/functions
   pnpm run dev
-
 ```
     
-
-Start frontend 
+### Step 5: Start the Frontend
+Navigate to the frontend directory and start the frontend server:
 
 ```bash
   cd ./packages/web
   pnpm run dev
 ```
-View the local website at http://localhost:3000
\ No newline at end of file
+
+### Step 6: Access the Local Website
+Once both the backend and frontend servers are running, you can view the application at:
+http://localhost:3000

```

Comment:
maybe add a section talking about how the project is part of ICSSC and links to the ICSSC github organization? Idk just an idea. Check Antalmanac's repo for reference to see what else we should add!